docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -1,171 +1,182 @@
 ---
-title: Finding and Choosing NuGet Packages
-description: An overview of how to find and choose the best NuGet packages for a project including details on the NuGet search syntax.
+title: Find and evaluate NuGet packages
+description: Find and evaluate publicly available NuGet packages for your project by using advanced nuget.org search filters and syntax.
 author: JonDouglas
 ms.author: jodou
-ms.date: 11/03/2021
+ms.date: 08/29/2022
 ms.topic: conceptual
 ---
 
-# Finding and evaluating NuGet packages for your project
+# Find and evaluate NuGet packages for your project
 
-When starting any .NET project, or whenever you identify a functional need for your app or service, you can save yourself lots of time and trouble by using existing NuGet packages that fulfill that need. These packages can come from the public collection on [nuget.org](https://www.nuget.org/packages/), or a private source that's provided by your organization or another third party.
+When you start a .NET project, or identify a functional need in your app or service, you can often install existing NuGet packages to save the time and trouble of [creating your own packages](../create-packages/overview-and-workflow.md). Existing packages can come from the [nuget.org](https://www.nuget.org/packages) public collection, or from private sources that your organization or another party provide.
```

Comment:
I only glanced over all the changes in this file. Maybe I missed it, but I didn't see anything to explicitly tell customers that all these advanced filtering options are server dependent. Especially because of the last sentence here, saying private sources, I think it's important to tell customers it might not work as expected.  NuGet.org only started supporting the syntax since last year, so there was 10 years where even nuget.org didn't support the syntax. So it's likely that many non-nuget.org package sources do not support any of these advanced syntaxes.

I don't have an idea on how to briefly say this, but hopefully you can come up with something. I hope that you also agree it's worth mentioning to customers.

Reply:
@zivkan - It's here, first sentences under Search syntax:

### Search syntax
Package search queries at nuget.org, from the NuGet CLI, and from within Visual Studio all use the same syntax. Other package sources, like Azure Artifacts or GitHub Package Repository, might use different syntax or might not support advanced filtering.

Do you think that's enough?

Reply:
Oops, sorry I missed that. After reviewing 5+ long docs PRs, I was rushing.  I had hoped to do some coding today 😁